pkgs/by-name/ke/keepassxc/package.nix

```diff
@@ -3,32 +3,27 @@
   stdenv,
   fetchFromGitHub,
   cmake,
-  qttools,
+  libsForQt5,
 
   apple-sdk_15,
   asciidoctor,
   botan3,
   curl,
   darwinMinVersionHook,
-  kio,
   libXi,
   libXtst,
   libargon2,
   libusb1,
   minizip,
+  nix-update-script,
   pcsclite,
   pkg-config,
   qrencode,
-  qtbase,
-  qtmacextras,
-  qtsvg,
-  qtx11extras,
   readline,
   wrapGAppsHook3,
-  wrapQtAppsHook,
   zlib,
 
-  LocalAuthentication,
+  darwin,
 
   withKeePassBrowser ? true,
   withKeePassBrowserPasskeys ? true,
```

Comment:
I meant, that we move the condition `stdenv.hostPlatform.isLinux` to here to replace the true's.